deploy/eck-operator/values.yaml

```diff
@@ -199,13 +199,13 @@ podMonitor:
   podMetricsEndpointConfig: {}
   # honorTimestamps: true
 
-# Internal use only
-internal:
+# Globals meant for internal use only
```

Comment:
Out of curiosity, why did you rename `internal` to `global`?

Reply:
Because `global` is a built-in thingy in Helm and I can share the values with the dependent chart. Otherwise we would have to set the `internal.k8sVersion` value again for the crd chart.